native/game_backend/src/game.rs

```diff
@@ -454,6 +455,19 @@ fn update_player_actions(players: &mut HashMap<u64, Player>, elapsed_time_ms: u6
     })
 }
 
+fn update_kill_counts(players: &mut HashMap<u64, Player>, killfeed: &Vec<KillEvent>) {
+    killfeed
+        .iter()
+        .for_each(|kill_event| match kill_event.kill_by {
+            EntityOwner::Player(player_id) => {
+                if let Some(player) = players.get_mut(&player_id) {
+                    player.add_kill();
+                }
```

Comment:
Should we also have a `player.add_death/0` ?

Reply:
No, I think we are not going to show the death count, and also players can't re-spawn